bftengine/tests/simpleTest/config_file_parser_test.cpp

```diff
@@ -0,0 +1,33 @@
+#include "config_file_parser.hpp"
+
+#include <iostream>
+#include <string>
+
+using std::cout;
+using std::cin;
+using std::string;
+
+int main(int argc, char **argv) {
+
+  cout << "Enter configuration file name\n";
+
+  string tmp, file = "config.txt";
+  cin >> tmp;
+  if (tmp == ".\n")
+    file = tmp;
+  concordlogger::Logger logger =
+      concordlogger::Logger::getLogger("simpletest.test");
+  ConfigFileParser parser(logger, file);
+  if (!parser.Parse())
+    return 1;
+  cout << "\n";
+  parser.Count("replicas_config");
```

Comment:
It would be nice if this test were a little stronger. It seems like it only verifies that parsing config.txt doesn't make the program crash. Can it also verify that the result of the parsing is what we expect?

Reply:
Done.